internal/xds/translator/listener.go

```diff
@@ -254,6 +254,10 @@ func (t *Translator) addXdsHTTPFilterChain(xdsListener *listenerv3.Listener, irL
 		return err
 	}
 
+	// Remove /healthcheck/fail from endpoints that will trigger a drain of listeners for better control
```

Comment:
this should probably live in `buildTCPListener` & `buildQUICListener`
https://github.com/envoyproxy/gateway/blob/72fadb7c1dbe7d660c2900527c34f53cfcfce7ac/internal/xds/translator/listener.go#L130

Reply:
that makes sense, addressed in afcbf23